product_docs/docs/pem/8/pem_rel_notes/index.mdx

```diff
@@ -6,11 +6,11 @@ The Postgres Enterprise Manager (PEM) documentation describes the latest version
 
 | Version                   | Release Date | Upstream Merges                                                                                                                                                                                                                                                                                                                                        | Accessibility Conformance                                                                           |
 | ------------------------- | ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | --------------------------------------------------------------------------------------------------- |
-| [8.5.0](03_850_rel_notes) | 23 Jun 2022  | NA                                                                                                                                                                                                                                                                                                                                                     | NA                                                                                                  |
+| [8.5.0](03_850_rel_notes) | 23 Jun 2022  | NA                                                                                                                                                                                                                                                                                                                                                     | [Conformance Report](https://www.enterprisedb.com/accessibility)                                    |
```

Comment:
I suggest we link to the specific report except for the link for 8.1.1 that doesn't have a specific report (I'm not sure why we link at all in that case though). The column header can be a link to https://www.enterprisedb.com/accessibility.

Reply:
I was linking it to a specific report previously but then changed it to this page, as this report gets published along with packages so we have to wait until it gets published. Still, if you think links to actual reports make more sense I can do that. Let me know